R/gtrends.R

```diff
@@ -94,13 +94,15 @@ gconnect <- function(usr = NULL, psw = NULL, verbose = FALSE) {
   
   authenticatePage2 <- getURL("http://www.google.com", curl = ch)
   
-  if (getCurlInfo(ch)$response.code == 200) {
+  if (grepl("The email or password you entered is incorrect.", authenticatePage)) {
```

Comment:
Can you check if this works for different local values, ie if you query in French (or even Danish :smiley:  )  ?